go/framework/bertybridge/bridge_test.go

```diff
@@ -121,8 +121,6 @@ func TestPersistenceProtocol(t *testing.T) {
 
 	const n_try = 4
 
-	testutil.FilterStabilityAndSpeed(t, testutil.Unstable, testutil.Slow)
```

Comment:
This test was failed due to race condition happen in multipeer-connectivity , so we can remove it. But let wait the test run on server.